src/parse-options.js

```diff
@@ -13,6 +17,13 @@ const DEFAULT_OPTIONS = {
 function parseOptions (providedOptions) {
   let options = Object.assign({}, DEFAULT_OPTIONS, providedOptions)
 
+  // normalizing options.color to be a boolean or a color value
+  options.color = (options.color === 'false' || options.color === false) ? false : options.color
```

Comment:
The switching conditions and ternary expressions make this a little hard for me to parse. What do you think about:

```javascript
if (options.color) {
  if (options.color === 'false') {
    options.color = false
  } else if (!(options.color in styles)) { // could also do if (!styles.hasOwnProperty(options.color))
    throw new Error('<our message>')
  }
}
```

There's no need to coalesce `false` or `undefined` to false, and we can handle the `'false'` case if `options.color` is actually defined.